src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/LicenseManager.cs

```diff
@@ -329,10 +327,8 @@ private static bool ValidateInternalRecursive(LicenseContext context, Type type,
                     if (license != null)
                     {
                         license.Dispose();
-                        license = null;
                     }
-                    string? temp;
```

Comment:
Actually, these `license = null` removals look potentially problematic.  Same case as the one the other day.  license is an out argument; if we set it to something, then dispose that something, and then an exception occurs, we don't want to have returned out a disposed license.  I think these nulls should be put back.

Reply:
There is only 1 place now with pragma, the rest stored licence into temporary local variables.